packages/vscode-extension/package.nls.json

```diff
@@ -559,24 +559,24 @@
 
   "teamstoolkit.walkthroughs.buildIntelligentApps.intelligentAppsIntroduction.title": "Introduction",
-  "teamstoolkit.walkthroughs.buildIntelligentApps.intelligentAppsIntroduction.description": "A declarative agent is a customized version of Microsoft 365 Copilot that allows users to create personalized experiences by declaring specific instructions, actions, and knowledge.\n\nThis guide provides information about how to build a declarative agent by using Teams Toolkit.",
-  "teamstoolkit.walkthroughs.buildIntelligentApps.intelligentAppsIntroductionWithChat.description": "A declarative agent is a customized version of Microsoft 365 Copilot that allows users to create personalized experiences by declaring specific instructions, actions, and knowledge.\n\nThis guide provides information about how to build a declarative agent by using Teams Toolkit.\n\nYou can also [chat with GitHub Copilot](command:fx-extension.invokeChat?%5B%22WalkThroughIntelligentAppsIntroduction%22%5D) to learn how to develop agent for Microsoft 365 Copilot.",
+  "teamstoolkit.walkthroughs.buildIntelligentApps.intelligentAppsIntroduction.description": "Build a declarative agent to customize Microsoft 365 Copilot with specific instructions, actions, and knowledge. Follow this guide to get started.",
+  "teamstoolkit.walkthroughs.buildIntelligentApps.intelligentAppsIntroductionWithChat.description": "Build a declarative agent to customize Microsoft 365 Copilot with specific instructions, actions, and knowledge. Follow this guide or [chat with GitHub Copilot](command:fx-extension.invokeChat?%5B%22WalkThroughIntelligentAppsIntroduction%22%5D) to get started.",
 
-  "teamstoolkit.walkthroughs.buildIntelligentApps.intelligentAppsEnvironment.title": "Get your environment ready",
-  "teamstoolkit.walkthroughs.buildIntelligentApps.intelligentAppsEnvironment.description": "To extend Microsoft 365 Copilot with declarative agent, in addition to Node.js and NPM requirements, you will also need a valid license to access Microsoft 365 Copilot. [Checkout this document for obtaining developer environment.](https://learn.microsoft.com/en-us/microsoft-365-copilot/extensibility/prerequisites#prerequisites)\n[Check Copilot License](command:fx-extension.checkCopilotAccess?%5B%22WalkThrough%22%5D)",
+  "teamstoolkit.walkthroughs.buildIntelligentApps.intelligentAppsEnvironment.title": "Set up your environment",
+  "teamstoolkit.walkthroughs.buildIntelligentApps.intelligentAppsEnvironment.description": "Set up your environment to build a declarative agent for Microsoft 365 Copilot. Ensure you have Node.js, NPM, and a valid Microsoft 365 Copilot license. Learn how to get a developer environment here. [Learn how to get a developer environment.](https://learn.microsoft.com/en-us/microsoft-365-copilot/extensibility/prerequisites#prerequisites)\n[Check Copilot License](command:fx-extension.checkCopilotAccess?%5B%22WalkThrough%22%5D)",
```

Comment:
Please remove the line - Learn how to get a developer environment here. It's a duplicate of hyperlinked line.

Reply:
Updated. Thanks!